pxd_io_uring.h

```diff
@@ -0,0 +1,160 @@
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
```

Comment:
Can we add a note on such header files indicating that they are included for build purpose only?